apps/website/src/content/docs/dropdownmenu.mdx

```diff
@@ -3,13 +3,12 @@ title: Dropdown menu
 description: The dropdown menu displays a list of selectable choices and options.
 propsPath: '@itwin/itwinui-react/esm/core/DropdownMenu/DropdownMenu.d.ts'
 thumbnail: Menu
+related:
+  - select
+  - combobox
+  - popover
```

Comment:
Are we not including Button any more?

Reply:
i'm not sure why we had button here, i changed it to more relevant components